src/components/Button/Button.tsx

```diff
@@ -13,6 +13,7 @@ interface ButtonProps {
   big?: boolean
   unstyled?: boolean
   onClick?: (event: React.MouseEvent<HTMLButtonElement>) => void
+  className?: string
```

Comment:
so this is just a suggestion but I think it will be useful for us to adopt throughout this lib... I recently discovered you can do this when defining prop types for components that implement HTML elements:
```
export const Button = (props: ButtonProps & React.HTMLAttributes<HTMLButtonElement>): React.ReactElement => {...
```

that will automatically extend any native HTML or React props that a `<button>` element should be able to receive, and then we don't have to explicitly include props such as `onClick` and `className` in our `ButtonProps`! super useful :D there is a full list of React types here https://www.saltycrane.com/cheat-sheets/typescript/react/latest/

Reply:
This sounds like a great idea, I'll update this PR accordingly.